ci-operator/jobs/openshift/multiarch/openshift-multiarch-release-4.5-periodics.yaml

```diff
@@ -1,7 +1,7 @@
 periodics:
 - agent: kubernetes
   cluster: api.ci
-  cron: 0 10,22 * * *
+  cron: 0 8,22 * * *
```

Comment:
It is according to the old schedule for 4.5, based on the new schedule it should look be 0 0,12 * * *